schema_utils.py

```diff
@@ -229,47 +261,113 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: list(str). An object.
+            min_value: int. The minimum value to check the length of object.
+
+        Returns:
+            bool. The given object has at least `min_value` elements.
```

Comment:
Always start "bool" descriptions with "Whether". Here: "Whether the given object has at least ...".

Ditto below.

Reply:
Done